Examples/MAX78000/TFT_Demo/example_config.h

```diff
@@ -34,6 +34,8 @@
 #ifndef EXAMPLES_MAX78000_TFT_DEMO_EXAMPLE_CONFIG_H_
 #define EXAMPLES_MAX78000_TFT_DEMO_EXAMPLE_CONFIG_H_
 
+#include "board.h"
```

Comment:
Wow, good catch!  "board.h" was never included, so the ifdefs below were never triggered...  missing `ENABLE_TS` entirely.